cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java

```diff
@@ -31,7 +27,8 @@
 public class RealtimeStageContext implements StageContext {
```

Comment:
how come this class is needed if there is a WorkerRealtimeContext or vice versa?